src/server/resolvers.ts

```diff
@@ -178,6 +187,17 @@ export const resolvers: Resolvers = {
 		note: async question => (await question).note || null,
 		prompt: async question => (await question).prompt,
 	},
+	Event: {
+		end: async event => (await event).end.getTime(),
```

Comment:
As an alternative to start and end times, we could do start and duration. It technically gives us the exact same information but might be more useful as display cares more about how long it is rather than the numerical time it ends which is a human construct. I realize we did end time last year which is where this came from, but it might be worth changing.

Reply:
It's also more maintainable for the organizer because more rescheduling will only need one value change.

Reply:
definitely start + duration